.github/workflows/nightly-benchmark.yml

```diff
@@ -0,0 +1,68 @@
+name: Nightly Benchmark
+
+on:
+  schedule:
+    - cron: "0 3 * * *"
+  workflow_dispatch:  # Allows manual trigger
+
+permissions:
+  contents: write
+  checks: write
+  pull-requests: write
```

Comment:
There is a duplicate `permissions` section - second under the only job: `benchmark`.